grpc/server.go

```diff
@@ -43,8 +44,27 @@ func (s *Server) StreamSearch(req *v1.SearchRequest, ss v1.WebserverService_Stre
 		return status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	onMatch := stream.SenderFunc(func(res *zoekt.SearchResult) {
-		ss.Send(res.ToProto())
+	onMatch := stream.SenderFunc(func(r *zoekt.SearchResult) {
+		result := r.ToProto()
+
+		f := func(filesChunk []*v1.FileMatch) error {
+			return ss.Send(&v1.SearchResponse{
```

Comment:
Note: 

If I am breaking up the search response into multiple separate messages, I probably need to ensure that the repoURLS and Linefragments mentioned in the invidiual messages contain entries that _only_ come from the chunk of file matches sent within that particular message (as opposed to the original big zoekt.SearchResult)

Reply:
Note: we don't use repoURLs or lineFragments in sourcegraph. I would consider removing them from the proto file to simplify this step.